server.js

```diff
@@ -70,17 +70,30 @@ app.get("/historical/:country", async function (req, res) {
   res.send(countryData);
 });
 
-app.get("/countries/:country", async function (req, res) {
+app.get("/countries/:search", async (req, res) => {
   let countries = JSON.parse(await redis.get(keys.countries));
-
-  const standardizedCountryName = countryMap.standardizeCountryName(req.params.country.toLowerCase());
-  let country = countries.find(e => {
-    // see if strict was even a parameter
-    if (req.query.strict) {
-      return req.query.strict.toLowerCase() == 'true' ? e.country.toLowerCase() === standardizedCountryName : e.country.toLowerCase().includes(standardizedCountryName)
-    }
-    else {
-      return e.country.toLowerCase().includes(standardizedCountryName);
+  const { search } = req.params, isText = isNaN(search);
```

Comment:
should be split into two assignments for readability

Reply:
sorry fixed